### ISSUE TITLE
Generalize the 'what-to-do' part after a subcommand completed

### ISSUE THREAD

**Issue by simonzhaoms** [mlhubber/mlhub#16]
There needs to a way to let model developer to describe what to do next after a command completed, and let the command line tool to present and format the what-to-do part.  The procedure needs to be generalised.

**Comment by simonzhaoms** [comment]
Compare
```
ml demo scatter-plot
```
and the suggestion for what-to-do after it (Note the `-r` part.)
```
ml print scatter-plot-r
```

The outputs are:
```console
$ ml demo scatter-plot

======================================== 
Introducing Scatter Plots with R ggplot2 
========================================

...
====================
Plot Weather Dataset
====================

... 
  $ ml print scatter-plot-r


$ ml print scatter-plot-r
mlhub: model 'scatter-plot-r' is not installed (/home/simon/.mlhub/).

Check for the model name amongst those installed:

  $ ml installed

Models can be installed from the ML Hub:

  $ ml install scatter-plot-r

Available pakages on the ML Hub can be listed with:

  $ ml available
```

**Comment by gjwgit** [comment]
The '-r' issue has been fixed in scatter-plot 1.1.2. Thanks.

**Comment by gjwgit** [comment]
The order of model specific commands is specified in DESCRIPTION.yaml. For the mlhub implemented commands the order is always: install, readme, configure, commands, demo. This is then followed by the commands specified in DESCRIPTION.yaml, in the same order, with mlhub printing the suggestion of the follow on command after each command is run. The text for the message comes from the command description in the DESCRIPTION.yaml file, formated as

```
To {}:

  $ ml <command> <model>
```
where {} is the text from the DESCRIPTION file and <command> and <model> are replaced appropriately.

When we get to the end, thank the user for using mlhub with the specific model.
